mypy/test/testdiff.py

```diff
@@ -53,6 +53,8 @@ def build(self, source: str) -> Tuple[List[str], Optional[Dict[str, MypyFile]]]:
         options.use_builtins_fixtures = True
         options.show_traceback = True
         options.cache_dir = os.devnull
+        options.python_version = (3, 6)
```

Comment:
That seems a bit arbitrary. Why not `mypy.defaults.PYTHON3_VERSION`? Or `sys.version_info[:2]`? (Then again I realize this is very far from your code -- the feature that this tests isn't even announced yet. :-)

Ditto for testmerge.py, and half-ditto for testsemanal.py.